usr/bin/playtron-factory-reset

```diff
@@ -52,9 +52,6 @@ function reset_etc {
 # change cwd to a directory that won't be deleted
 cd /
 
-# clear all package changes
-rpm-ostree reset &> /dev/null
```

Comment:
Before I forget, we probably want to keep this. If `rpm-ostree` is installed and there are layered packages or other overrides, it will break `bootc`.

Perhaps we can add a bit of logic to see if the binary even exists first. If it does, then perform a reset.

```
if [ -f /usr/bin/rpm-ostree ]
    then rpm-ostree reset &> /dev/null
fi
```